pkg/controller/stacks/install/installjob.go

```diff
@@ -265,23 +263,9 @@ func (jc *stackInstallJobCompleter) createJobOutputObject(ctx context.Context, o
 		}
 	}
 
-	// We want to clean up any installed CRDS when we're deleted. We can't rely
-	// on garbage collection because a namespaced object (StackInstall) can't
-	// own a cluster scoped object (CustomResourceDefinition), so we use labels
-	// instead.
+	// TODO(displague) parentlabels can only express a single parent, CRDs
```

Comment:
Per our conversation, I'm going to not add the "single parent" labels on CRDs here.  We will continue to use the other resources that unpack handles (Stack, StackDefinition).  We may revisit the use of single parent labels in that scenario later.

I'm also going to remove the existing "single parent" labels on StackInstall, to avoid leaking labels.  The StackInstall controller is the referent. When a StackInstall is being deleted it can search for any CRDs that match its single-parent labels (similar to the DeleteAll CRD call that is being removed by this PR) and then remove that set of labels.

This can be optimized to only take place after CRDs lacking multiple-parent labels are deleted.  This label cleanup code can be removed in the future since future stackinstalled CRDs will not contain this set of single parent labels.